database/sampleDataQueries.sql

```diff
@@ -13,23 +13,14 @@ VALUES
+  ('River', 'Dog', 'Mixed', 'Male', 4, 'Medium, 25lbs', 1, '/images/petimage-testMix.png', 'Available', '2022-01-27 12:32:05', 0, 1, 0, 1, 1, 1,
    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit facilisis elit, id volutpat dolor porttitor sed. Proin erat diam, venenatis ac lacinia a, condimentum ac turpis. Fusce eu nisl tempor, feugiat neque vitae, tincidunt est.'),
-  ('Adeline', 'Cat', 'Domestic Shorthair', 'Female', 2, 'Small, 8.8lbs', 2, '', 'Available', '2022-01-27 12:32:05', 0, 0, 0, 1, 1, 1,
+  ('Adeline', 'Cat', 'Domestic Shorthair', 'Female', 2, 'Small, 8.8lbs', 2, '/images/petimage-testCat.png', 'Available', '2022-01-27 12:32:05', 0, 0, 0, 1, 1, 1,
    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit facilisis elit, id volutpat dolor porttitor sed. Proin erat diam, venenatis ac lacinia a, condimentum ac turpis. Fusce eu nisl tempor, feugiat neque vitae, tincidunt est.'),
-   ('CHIP', 'Dog', 'Chihuahua', 'Male', 1, 'Small, 5lbs', 1, '', 'Pending', '2022-01-27 12:32:05', 1, 1, 0, 1, 1, 1,
-   'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit facilisis elit, id volutpat dolor porttitor sed. Proin erat diam, venenatis ac lacinia a, condimentum ac turpis. Fusce eu nisl tempor, feugiat neque vitae, tincidunt est.'),
-  ('COOPER', 'Dog', 'Mixed', 'Male', 4, 'Medium, 25lbs', 2, '', 'Available', '2022-01-27 12:32:05', 0, 1, 0, 1, 1, 1,
-   'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit facilisis elit, id volutpat dolor porttitor sed. Proin erat diam, venenatis ac lacinia a, condimentum ac turpis. Fusce eu nisl tempor, feugiat neque vitae, tincidunt est.'),
-  ('Choco', 'Cat', 'American Shorthair', 'Male', 9, 'Small, 13lbs', 1, '', 'Pending', '2022-01-27 12:32:05', 0, 0, 0, 1, 1, 1,
-   'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit facilisis elit, id volutpat dolor porttitor sed. Proin erat diam, venenatis ac lacinia a, condimentum ac turpis. Fusce eu nisl tempor, feugiat neque vitae, tincidunt est.'),
-   ('Rab', 'Rabbit', 'Lionhead rabbit', 'Male', 2, 'Small, 4lbs', 1, '', 'Available', '2022-01-27 12:32:05', 0, 0, 0, 0, 0, 0,
-   'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit facilisis elit, id volutpat dolor porttitor sed. Proin erat diam, venenatis ac lacinia a, condimentum ac turpis. Fusce eu nisl tempor, feugiat neque vitae, tincidunt est.'),
-   ('Ham', 'Hamster', 'Syrian hamsters', 'Female', 2, 'Small, 3lbs', 2, '', 'Pending', '2022-01-27 12:32:05', 0, 0, 0, 0, 0, 0,
-   'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed blandit facilisis elit, id volutpat dolor porttitor sed. Proin erat diam, venenatis ac lacinia a, condimentum ac turpis. Fusce eu nisl tempor, feugiat neque vitae, tincidunt est.');
-
+  ('Rab', 'Rabbit', 'Lionhead rabbit', 'Male', 2, 'Small, 4lbs', 1, '/images/petimage-testRabbit.png', 'Available', '2022-01-27 12:32:05', 0, 0, 0, 0, 0, 0,
```

Comment:
I think the format of the species/breed needs to be 'Other' and 'Rabbit' ('or 'Lionhead rabbit')

Reply:
I completely forgot that we decided to use other! Thanks!